src/styles/theme.scss

```diff
@@ -5,6 +5,7 @@
 
 /** System theme (auto-detects OS light/dark mode), other themes extend this one */
 :root {
+  --spacing-nano: 0.125rem;
```

Comment:
We did extensive research to come up with the upper and lower values of the spacing. What we usually do if we want to go below `micro` spacing (very rare cases) is to use [`calc`](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) e.g. `calc(0.5 * var(--spacing-micro))`